<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1GINDK7" failures="3" tests="13" name="TestNG Concept" time="0.261" errors="0" timestamp="2024-07-04T16:54:19 IST">
  <testcase classname="testNGBasics.TestRetry" name="drive" time="0.036">
    <skipped/>
  </testcase> <!-- drive -->
  <testcase classname="testNGBasics.TestRetry" name="drive" time="0.007">
    <skipped/>
  </testcase> <!-- drive -->
  <testcase classname="testNGBasics.TestRetry" name="drive" time="0.004">
    <skipped/>
  </testcase> <!-- drive -->
  <testcase classname="testNGBasics.TestRetry" name="drive" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGBasics.TestRetry.drive(TestRetry.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- drive -->
  <testcase classname="testNGBasics.TestRetry" name="ride" time="0.001">
    <skipped/>
  </testcase> <!-- ride -->
  <testcase classname="testNGBasics.TestRetry" name="ride" time="0.001">
    <skipped/>
  </testcase> <!-- ride -->
  <testcase classname="testNGBasics.TestRetry" name="ride" time="0.004">
    <skipped/>
  </testcase> <!-- ride -->
  <testcase classname="testNGBasics.TestRetry" name="ride" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGBasics.TestRetry.ride(TestRetry.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ride -->
  <testcase classname="testNGBasics.TestRetry" name="swim" time="0.005">
    <skipped/>
  </testcase> <!-- swim -->
  <testcase classname="testNGBasics.TestRetry" name="swim" time="0.001">
    <skipped/>
  </testcase> <!-- swim -->
  <testcase classname="testNGBasics.TestRetry" name="swim" time="0.002">
    <skipped/>
  </testcase> <!-- swim -->
  <testcase classname="testNGBasics.TestRetry" name="swim" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGBasics.TestRetry.swim(TestRetry.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- swim -->
  <testcase classname="testNGBasics.TestRetry" name="walk" time="0.003"/>
</testsuite> <!-- TestNG Concept -->
